website/docs/documentation/Pipelines/Smart-seq2_Single_Sample_Pipeline/README.md

```diff
@@ -66,10 +68,12 @@ The Smart-seq2 Single Sample workflow requires multiple reference indexes. Infor
 
 | Workflow Step | Reference name | Reference Description | Tool |
 | --- | --- | --- | --- |
-| All | input_id | Sample name or Cell ID | NA |
+| All | input_id | String containing sample name or cell ID. It can be a human-readable descriptor (i.e. "cell1") or a UUID | NA |
```

Comment:
If we describe it as cell_names rather than ID it is more consistent.

Reply:
Thank you! I removed sample and incorporated "name".